cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -379,8 +385,7 @@ def export_status_handler(request, course_key_string):
                 'response-content-type': 'application/x-tgz'
             })
         else:
-            # local file, serve from the authorization wrapper view
-            output_url = reverse_course_url('export_output_handler', course_key)
+            output_url = artifact.file.storage.url(artifact.file.name)
```

Comment:
@xitij2000, can you explain for what file storage this branch will be executed?

Reply:
@afzaledx I think this is our main use case. We use SWIFT for storage, using the [django-swift-storage](https://github.com/dennisv/django-storage-swift) library. In the old code, it just checks if the ``bucket`` attribute exists on the storage class (which is S3-specific). The [SwiftStorage](https://github.com/dennisv/django-storage-swift/blob/d7c0e8a256c241f429c464c487a14e21629acd7a/swift/storage.py#L118) class doesn't have this property so on our instances it ends up using the default FileSystem-specific exporter. 

So this particular line is for the case when someone is using a custom storage class which is neither FileSystemStorage nor the S3 storage class (for instance SwiftStorage). I assume this is the behaviour any person would want if they are not using S3 or FileSystemStorage.